src/components/localstorage/localStorageReader.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import FileSaver from "file-saver";
+
+const mapStateToProps = state => ({
+  items: state.localStorage.items
+});
+
+const mapDispatchToProps = dispatch => ({
+});
+
+function generateFileLocalStorageFn() {
+  let output = "";
+  const localStorageItems = JSON.parse(localStorage.getItem("reduxPersist:localStorage"));
+  const items =  localStorageItems.items.map((e, i) => (e));
+  Object.keys(items).forEach( (key) => {
+    output += items[key] + "\n";
+  });
+  let blob = new Blob([output], {type: "text/plain;charset=utf-8"});
+  FileSaver.saveAs(blob, "localStorage.txt");
+}
+
+function generateFilePropsFn(itemsProps) {
+  let output = "";
+  const items =  itemsProps.map((e, i) => (e));
+  Object.keys(items).forEach((key) => {
+    output += items[key] + "\n";
+  });
+  let blob = new Blob([output], {type: "text/plain;charset=utf-8"});
+  FileSaver.saveAs(blob, "localStorage.txt");
+}
```

Comment:
these should go in an epic. look at src/epics/
the flow would be:
1. fire a redux action (e.g. DOWNLOAD_HANDLES)
2. intercept it using an epic
3. do the `generateFilePropsFn` stuff